packages/common/web/statics/styles/fonts.scss

```
@@ -97,6 +97,15 @@ $text-decorations: (
   min-height: var(--ricos-custom-#{$name}-min-height, map-get($min-heights, $name));
   @if $name == 'p' {
     color: $ricos-p-color;
+    @if $isViewer {
+      @at-root #{'& > * > span'} {
+        background-color: $ricos-p-background-color;
+      }
+    } @else {
+      @at-root #{'& > span'} {
```

Shouldn't it be only the first span in the dom?
For example, 

```suggestion
      @at-root #{'& + span'} {
```